src/redis_cmd.cc

```diff
@@ -3450,13 +3467,21 @@ class CommandSlaveOf : public Commander {
       if (s.IsOK()) {
         *output = Redis::SimpleString("OK");
         LOG(WARNING) << "MASTER MODE enabled (user request from '" << conn->GetAddr() << "')";
+        if (svr->GetConfig()->cluster_enabled) {
+          svr->slot_migrate_->SetMigrateStopFlag(false);
+          LOG(INFO) << "Change server role to master, restart migration task";
+        }
```

Comment:
isn’t this block of code are dead code? since in the beginning we will throw an error if we are in the cluster mode

sorry for commenting in this very old PR... i tracked the changes, afraid i am missing something.